src/common/api/hive-engine.ts

```diff
@@ -188,3 +188,35 @@ export const getMarketData = async (symbol: any) => {
   });
   return history;
 };
+
+export async function getTransactions(symbol: string, account: string, limit: number, offset?: number): Promise<any> {
+  const url: any = engine.mainTransactionUrl;
+  return axios({
+    url: url,
+    method: "GET",
+    params: {
+      account,
+      token: symbol,
+      limit,
+      offset
+    }
+  }).then((response) => {
+    return response.data;
+  });
+};
+
+export async function getOtherTransactions(account: string, limit: number, symbol: string, offset: number = 0) {
+  const url: any = engine.otherTransactionsUrl;
+  const response = await axios({
+    url: url,
```

Comment:
You may leave just `url` instead of `url: url`

Reply:
oh, that's true,
 Thanks🙏

Reply:
Still didn't fixed